frontend/src/__mocks__/mockNotebookK8sResource.ts

```diff
@@ -0,0 +1,298 @@
+import { NotebookKind } from '~/k8sTypes';
+
+type MockResourceConfigType = {
+  name?: string;
+  displayName?: string;
+  namespace?: string;
+  user?: string;
+  description?: string;
+};
```

Comment:
For knowledge -- `Partial<MyType>` will make everything optional. It's an alternative to making every item have a  `?`.

Reply:
yeah probably should.